web/src/app/admin/admin-outgoing-logs/OutgoingLogsControls.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react'
+import { Grid, Typography, IconButton } from '@mui/material'
+import makeStyles from '@mui/styles/makeStyles'
+import RestartAltIcon from '@mui/icons-material/RestartAlt'
+import { theme } from '../../mui'
+import { ISODateTimePicker } from '../../util/ISOPickers'
+import { useResetURLParams, useURLParam } from '../../actions'
+import Search from '../../util/Search'
+import { LOAD_AMOUNT } from './OutgoingLogsList'
+
+interface Props {
+  totalCount: number
+}
+
+const useStyles = makeStyles<typeof theme>({
+  filterContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  resetButton: {
+    height: 'min-content',
+    alignSelf: 'center',
+  },
+})
+
+export default function OutgoingLogsControls(p: Props): JSX.Element {
+  const classes = useStyles()
+
+  const [start, setStart] = useURLParam<string>('start', '')
+  const [end, setEnd] = useURLParam<string>('end', '')
+  const [key, setKey] = useState(0)
+  const resetDateRange = useResetURLParams('start', 'end')
+  const [limit] = useURLParam<string>('limit', '1')
+  const _limit = parseInt(limit, 10)
```

Comment:
We should invert this, the convention is to use the `_` prefix as temporary meaning "don't use" and the non-prefixed in the rest of the function body.

Better yet, passing `1` instead of `'1'` as the default value should cause `useURLParam` to return an int without parsing. It will convert the parameter to whatever type is passed in as default.

Reply:
I went in and inverted that. I'm getting an error using a number type for the useURLParam. Looks fixable but likely a bug in the hook.

<img width="483" alt="Screen Shot 2021-12-28 at 11 28 28 AM" src="https://user-images.githubusercontent.com/28174559/147591739-af9e4d65-7ffc-48a2-8f38-64ed53324ccb.png">
m

Reply:
Yeah, it was only returning a string when I implemented which is why I had everything converting to strings at the moment. As for the `_limit` i did that so i didn't have to rename the urlParam to be different, but makes sense to use the standard convention anyways.